Libplanet.Net/IceServer.cs

```diff
@@ -22,7 +22,11 @@ public IceServer(Uri url)
             Func<string, string, bool, char, (string, string, bool)> parser =
                 (username, credential, isUsernameChar, c) =>
                     c == ':'
-                        ? (username, credential, !isUsernameChar)
+                        ? isUsernameChar
+                            ? (username, credential, !isUsernameChar)
+                            : throw new ArgumentException(
+                                $"Uri.UserInfo '{url.UserInfo}' have to contain single colon " +
+                                "as a delimiter between username and credential.")
```

Comment:
Using something like `int colonCount` instead of `bool isUsernameChar` might make this more readable. 🙂

Reply:
@greymistcube Updated PR according to your comments :)

Reply:
Actually, you can remove `isUsernameChar` argument. It's redundant information with `colonCount`. 😗

Reply:
@greymistcube I think it's better to use `bool isBeforeColon` instead of `int colonCount`, since it will have only 0~1 range, any opinion for this?

Reply:
Either only using `bool colonCounted` or `int colonCount` is both fine I guess. Now that I look at it, perhaps `isUsernameChar` was a bad name. 😐